cli/kubectl_kadalu/storage_add.py

```diff
@@ -15,8 +15,8 @@ def storage_add_args(subparsers):
     parser_add_storage.add_argument(
         "--type",
         help="Storage Type",
-        choices=["Replica1", "Replica3"],
-        default="Replica1"
+        choices=["Replica1", "Replica3", "External"],
+        default=None
```

Comment:
If default is `None` then can we make this mandatory. `required=True` makes it mandatory.

Reply:
No, I don't it mandatory. Idea was if --external is used, I will set this to External, but if there are other things, and no type, then it will be set to Replica1. Because if it picks 'Default', I won't know in --external validation.